src/platform/backends/lxd/lxd_vm_image_vault.cpp

```diff
@@ -421,17 +432,26 @@ void mp::LXDVMImageVault::lxd_download_image(const QString& id, const QString& s
     source_object.insert("server", stream_location);
     source_object.insert("protocol", "simplestreams");
     source_object.insert("image_type", "virtual-machine");
+    source_object.insert("fingerprint", id);
 
-    if (id.startsWith(release))
-    {
-        source_object.insert("fingerprint", id);
-    }
-    else
+    QJsonObject image_object{{"source", source_object}};
+
+    auto release = QString::fromStdString(query.release);
+
+    if (!id.startsWith(release))
     {
-        source_object.insert("alias", release);
-    }
+        QJsonObject properties_object{{"query.release", "default"},
+                                      {"query.remote", QString::fromStdString(query.remote_name)}};
 
-    QJsonObject image_object{{"source", source_object}};
+        // Need to save the original image's last_used_at as a property since there is no way to modify the
+        // new image's last_used_at field.
+        if (!last_used.isEmpty())
+        {
+            properties_object.insert("last_used_at", last_used);
+        }
```

Comment:
Can't we rely on `last_used_at` being empty until it's used?

Reply:
Which `last_used_at`?  The one LXD stores in the main image object or this one here I'm introducing?